redcap_bridge/utils.py

```diff
@@ -81,3 +81,27 @@ def remove_columns(csv_file, compressed_file=None):
 def exportCSVtoXLS(csv_file, compressed_file=None):
     read_file = pd.read_csv(csv_file, na_filter=False, dtype='str')
     read_file.to_excel(r'Path', index=None, header=True)
+
+def conversion_to_odml_table_descriptor(full_elabbook_csv, session_number):
```

Comment:
Can you add a keyword argument to automatically save this to a csv file if a filename is provided?